plugins/wallabag.koplugin/main.lua

```diff
@@ -695,6 +763,36 @@ function Wallabag:setFilterTag(touchmenu_instance)
     self.tag_dialog:onShowKeyboard()
 end
 
+function Wallabag:setIgnoreTags(touchmenu_instance)
+   self.ignore_tags_dialog = InputDialog:new {
+        title =  _("Set a comma separated list of tags"),
```

Comment:
```suggestion
        title =  _("Tags to ignore"),
```

I would put the instruction in the description instead.